runtime/src/bank.rs

```diff
@@ -464,16 +473,23 @@ impl Bank {
         ancestors
     }
 
+    /// computed unix_timestamp at this slot height
+    pub fn unix_timestamp(&self) -> i64 {
+        self.genesis_creation_time
+            + ((self.slot as u128 * self.ns_per_slot) / 1_000_000_000_000) as i64
```

Comment:
Sure you have the right amount of zeros there? At a glance, it looks like 3 too many.